docs/post-pivot-configuration.md

```diff
@@ -0,0 +1,71 @@
+# Cluster configuration
+
+- [IBU/I post pivot configuration](#ibi-post-pivot-configuration)
+  - [Overview](#overview)
+  - [Network configuration](#network-configuration)
+  - [Recertification flow](#recertification-flow)
+  - [User specifications](#user-specifications)
+
+## Overview
+
+Post pivot configuration is on charge for applying user configurations after rebooting in new stateroot.
+- Perform networking configuration
+- Recertification flow
+- User specifications:
+  - Run recert flow
+  - Apply new hostname
+  - Apply new ip
+  - Apply all user specific configurations that user provides as https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/clusterinfo/clusterinfo.go#L8
+  - Applying extra manifests
+
+## Mount configuration folder
+There are 2 differences ways to provide configuration folder to:
+1. Add it as part of stateroot configuration process as we do in IBU, should be placed in /opt/openshift 
+2. Provide it as block device with label "relocation-config" , in this case we will mount it.
+
+// TODO: describe folder structure
+Folder structures can be seen https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/installation_configuration_files/scripts/prepare-installation-configuration.sh#L30 
+and will be described later on
+
+## Network configuration
+
+User can provide specific network configuration that he wants to be applied on the node as part of installation process.
+Currently, we support nmconnection files that should be placed in /opt/openshift/network-configuration/system-connections
+Prepare installation service is on charge to copy them to network manager configuration folder and restart NM
+https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/installation_configuration_files/scripts/prepare-installation-configuration.sh#L54
```

Comment:
nit: I think these links could be relative paths, which would avoid the explicit repo and branch reference. ie.
`[prepare-installation-configuration.sh](installation_configuration_files/scripts/prepare-installation-configuration.sh#L54)`

Reply:
Also, please use the markdown syntax as Don suggested
[Link text](URL/relative path)